tests/test_feedexport.py

```diff
@@ -1320,96 +1323,271 @@ def test_extend_kwargs(self):
             data = yield self.exported_data(items, settings)
             self.assertEqual(row['expected'], data[feed_options['format']])
 
+
+class FeedPostProcessedExportsTest(FeedExportTestBase):
+    __test__ = True
+    items = [
+        {'foo': 'bar1', 'baz': ''},
+        {'egg': 'bar2', 'baz': 'quux'},
+    ]
```

Comment:
I don’t _think_ multiple items and multiple fields can affect post-processing. So, to keep things minimal, maybe it would make sense to keep a single item with a single field.